### DISCUSSION TITLE
Exiting with status '14' (+ log file)

### DISCUSSION THREAD

**Discussion by BazZzilio** [winapps-org/winapps#348] [Support]
Bazzite 41.
Very weird issue. I can easily connect to the RDP (xfreerdp /v:192.168.122.155 /u:RDPWindows /p:xxxxx) but when im trying to use ./setup.sh Im getting screenshot number 1. On screenshot number 2 the issue itself. How can i fix it?
![Terminal](https://github.com/user-attachments/assets/6322dec8-b935-458e-888d-3e91fe12fa49)
![Log](https://github.com/user-attachments/assets/d7fa93d2-8bf9-496f-8148-39e8b6f7d499)

**Comment by SergioLoaiza** [comment]
Hi, I got the same error; any hint of how to resolve it?

**Comment by oskardotglobal** [comment]
There is no official discord channel.

**Comment by mattcbro** [comment]
I have the same problem.  I put my user name and password into winapps.conf.  After running the install script I actually see the vm window appear and attempt to login to my user name,  but then it just hangs and the install fails.

I can also use rdp to log into the vm from the command line via:
```xfreerdp /u:matt /p:mypassword /v:127.0.0.1```

**Comment by mattcbro** [comment]
The problem appears to be the recent freerdp v 3.9.  While I can use it to log into my docker vm,  I can not launch programs in the vm using the externa freerdp with the /app flag.  Moreover the clipboard doesn't work and several other things just don't work,  such as window resizing.  I might just go back to using vmware player if I can't fix this.

**Comment by oskardotglobal** [comment]
@mattcbro Is it possible that you're running Podman with Win11?
